### DISCUSSION TITLE
The documentation shoud introduce `ctx`

### DISCUSSION THREAD

**Discussion by pplmx** [redis/go-redis#1985] [Q&A] [answered]
Here, it just says you should use `ctx`, but not say what it is.
The whole doc also doesn't say what it is(Or only I don't find it?).
![image](https://user-images.githubusercontent.com/26994103/148335198-d23a550c-6693-4713-8289-bb3cd442f116.png)

**Comment by 631086083** [comment, marked as answer]
context.Context is the interface that the Go language introduced to the standard library in version 1.7. you can use google to search it

**Reply by pplmx** [reply]
I know it. I just want to say the documentation directly use `ctx` and does not declare it, which is confusing for the beginners personally.
The documentation should declare it before using it, which is once at least in the doc. Currently, the whole doc doesn't have this.
